### ISSUE TITLE
Export `SummarizedExperiment` object

### ISSUE THREAD

**Issue by csoneson** [csoneson/rnaseqworkflow#4]
Export `SummarizedExperiment` object (with data and DGE results) for use with e.g. iSEE.

**Comment by csoneson** [comment]
Probably should be a `SingleCellExperiment` object, to include dimension reduction results.

**Comment by fionarhuang** [comment]
The corresponding work is updated in the branch exportSE_fiona.
To summarize the work, I have created the singleCellExperiment (sce) and SummarizedExperiment  (se) in the following way:

**The original output from the prepare_results_for_shiny.R is as below.**
list(wideResults = list(edgeR = edgeRwide), 
             longResults = list(edgeR = edgeRlong), 
             geneModels = genemodels,
             bwFiles = bwfiles, 
             bwCond = condition, 
             dimReds = list(MDS = mds),
             abundances = list(logCPM = logcpms), 
             geneInfo = genes)

I have created the sce and se to store the result as below.
Assays: 
a list of two tables: 
raw count (data saved from run_dge_edge.R)
count in logCPM (apply cpm to the raw count with log =TRUE and prior.count = 2; actually, it is the matrix form of value column in abundances)

colData: cbind(bwFiles,  bwCond)

rowData: 
Gene, symbol, gene_biotype, chromosome, strand
These columns are from tx2gene and the row orders are the same to the rownames of the count table (data saved from run_dge_edge.R)
type, group
These columns are obtained from mds
Output from edgeR
The result table of each contrast is saved as a column in the rowData with the name of the contrast. So, actually, it's a DataFrame in a column.

Metadata: list(geneModels, geneInfo)

ReducedDims: (only exists in sce)
Mds without ID type and group columns
